Styleguide/Views/Styleguide/MvcPartialComponent.cshtml

```diff
@@ -1,11 +1,8 @@
-﻿@using System.Web.Mvc
-@using System.Web.Mvc.Html
+@using Forte.Styleguide.Views.Styleguide
```

Comment:
can't we do as before with model:

`@model MvcPartialComponentViewModel`

and then use 
`Model.LayoutPath`

instead of 
`(MvcPartialComponentViewModel)ViewData["Model"])`
?

Reply:
It doesn't work then. We were trying to solve that problem with Marek yesterday and only this solution worked.